brainstormforce/ultimate-addons-for-gutenberg
# SPCTR-812 Fix: Title Margin not getting applied for vertical style presets

### Description
Tabs block title Margin option not working for vertical-style Presets. 

### Types of changes
Bug fix (non-breaking change which fixes an issue)


### How has this been tested?

1. Add Tabs block.  
2. Select Third or Forth Preset  
3. Go to selected Block Settings -> Style tab → Title → Normal → Typography Section.
4. Try to set a Margin.
5. Margin should get applied in the preview and front end both. 

### Checklist:
